shim/src/scopeHandler.js

```diff
@@ -26,17 +26,13 @@ const alwaysThrowHandler = new Proxy(freeze({}), {
  * - hide the unsafeGlobal which lives on the scope chain above the 'with'.
  * - ensure the Proxy invariants despite some global properties being frozen.
  */
-export function createScopeHandler(unsafeRec) {
+export function createScopeHandler(unsafeRec, safeGlobal = {}) {
```

Comment:
We should talk about whether this should have a safeGlobal arg, whether it should default to anything, and whether {} is the right default.